packages/rum/src/domain/record/viewports.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { isSafari } from 'packages/core/test/specHelper'
```

Comment:
I am not sure to understand everything in `viewport.ts` but there is a lot of complexity and only two unit tests. Are you sure to have covered every case?

Reply:
It's true we don't have tests covering the _new_ pinch zoom + pinch scroll functionality. Unfortunately our unit tests don't expose hooks to the automation library, and our E2E tests need some large changes to synthetically trigger pinch zoom and page zoom while still being able to unit test inside the codebase.

So for this PR, under the safety of a feature flag we are testing to make sure we don't break existing functionality- specifically innerWidth and scroll.